src/components/graph/Graph.tsx

```diff
@@ -255,17 +249,12 @@ export const Graph: FC<IGraphProps> = (props: IGraphProps) => {
   useEffect(() => {
     const zoomSelection: Selection = d3.select(`#${graphContainerId}`);
     zoomSelection.call(zoomRef.current);
+    if (!graphConfig.zoom.zoomByScroll) {
+      zoomSelection.on("wheel.zoom", null);
+    }
     const behavior = getGraphBehavior(props.behaviorRef);
-    behavior?.setupZoomBehavior(
-      zoomSelection,
-      zoomRef,
-      props.config?.zoom?.disableScrollToZoom
-    );
-  }, [
-    graphContainerId,
-    props.behaviorRef,
-    props.config?.zoom?.disableScrollToZoom
```

Comment:
Should not rely on original config. It's not merged and normalized with default config.